assets/training/model_evaluation/components/distributed_model_prediction/spec.yaml

```diff
@@ -1,6 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
 name: model_prediction_with_container 
-version: 0.0.1
+version: 7.25.2024.16
```

Comment:
Are we introducing a new version scheme?

Reply:
Yes, it was accidentally added. It was the version I was using for debugging. I have fixed this